src/Uno.UWP/ApplicationModel/DataTransfer/DragDrop/Core/CoreDragInfo.cs

```diff
@@ -0,0 +1,111 @@
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Threading;
+using Windows.Devices.Input;
+using Windows.Foundation;
+using Windows.System;
+using Uno.Extensions;
+using Uno.Logging;
+
+namespace Windows.ApplicationModel.DataTransfer.DragDrop.Core
+{
+	internal interface IDragEventSource
+	{
+		long Id { get; }
+
+		uint FrameId { get; }
+
+		//bool IsExternal { get; }
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6334034)